lib/bap_traces/bap_trace_meta.ml

```diff
@@ -41,7 +41,7 @@ module Binary = struct
        @\nmd5sum: %s\
        @\n@[<2>\
        args:   %a@]\
-       @\n@[<2>envp:@\n%a@]@]@\n}" t.path (Digest.to_hex t.md5sum) Args.pp t.args Envp.pp t.envp
+       @\n@[<2>envp:@\n%a@]@]@\n}" t.path t.md5sum Args.pp t.args Envp.pp t.envp
```

Comment:
This will print digest as a string of bytes instead of a nice and common to us hex representation of the MD5sum

Reply:
Are you sure? `t.md5sum` is already a string. (The value `t` here is not of type `Md5_lib.t`.)

Reply:
Yep, it is a `string`, but not a string of ASCII characters, but a string of bytes. The `string` type is not constrained only to printable characters, the only difference between strings and bytes is that the former is non-mutable. So in the trace `md5sum` is stored as 32 bytes of data, not in a textual and printable representation.